backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/process/GenerateDeliveryDays.java

```diff
@@ -58,18 +58,19 @@
 	private Timestamp p_DateFrom = null;
 	private Timestamp p_DateTo = null;
```

Comment:
as written in the channel, i'm unsure about the role of `p_DateFrom` and `p_DateTo` in case a  p_tourVersion is selected. 
Maybe you should preset them? see `IProcessDefaultParametersProvider`

Reply:
Clarified with Mark, we shall leave the behaviour as it is in the tour window. Parameters are set and overwritten if necessary